ChronoVisor/src/ChronicleMetaDirectory.cpp

```diff
@@ -251,43 +271,44 @@ int ChronicleMetaDirectory::acquire_story(const std::string &client_id,
     auto chronicleMapRecord = chronicleMap_->find(cid);
     if (chronicleMapRecord != chronicleMap_->end()) {
         Chronicle *pChronicle = chronicleMapRecord->second;
-        /* Then check if Story exists, fail if false */
+        /* Then check if Story already_acquired_by_this_client, fail if false */
         uint64_t sid = pChronicle->getStoryId(story_name);
         if (sid == 0) {
+            LOGD("StoryID=%lu name=%s does not exist", sid, story_name.c_str());
```

Comment:
same comment as on lines 115-132.   use the client map in Story  in favor of global multimap